packages/react-scripts/fixtures/yarn-ws/ws/packages/comp2/index.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Comp2 from './';
```

Comment:
Would referencing its own package name work here? e.g. `from 'comp2';`

Reply:
Yes, but very indirectly, only because it's able to pick up the hoisted version of itself from the top-level (shared) node_modules.  (That's because, in both yarn WS and lerna, packages can pick up other packages' hoisted dependencies.)  So, I think './' is correct, but maybe there's something more you're getting at here?

Reply:
I suppose I'm saying that we might as well test that instead

Reply:
One notable difference between YW and Lerna is that YW hoists the symlinks to the root, and Lerna _does not_, making relative symlinks between sibling package directories. The node resolution is ultimately identical, when consumed as a dependency.

However, this is different for tests. When comp2's `main` is `index.js`, `import Comp2 from '.';` is correct. Using the package name is not appropriate for a test import.

Reply:
Good to know, thanks.

Reply:
> One notable difference between YW and Lerna is that YW hoists the symlinks to the root, and Lerna does not

Ok, so we should definitely e2e test w/ lerna since it can be a little different.

PS, --hoist basically makes lerna behave the same as YW, right?  Or maybe we should test that, too, to make sure.